.cicd/create-docker-from-binary.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo ":docker::package: Push EOS ubuntu 18.04 base Image to Dockerhub"
+
+# buildkite-agent artifact download someversion.deb .
```

Comment:
Let's remove this comment.

Reply:
ok, understood and done!

Reply:
The comment is still here it seems

Reply:
Sorry I changed the wrong line, now it's done